#!/usr/bin/python3
# function that raises a name exception with a message
# using def raise_exception_msg(message="") prototype

def raise_exception_msg(message=""):
    raise NameError(message)
